.gitignore

```diff
@@ -61,3 +61,4 @@ node_modules/prettier/plugins/yaml.mjs
 package-lock.json
 git
 node_modules/
+__pycache__
```

Comment:
This change is not related to adding testing documentation.

Reply:
This change has now been removed.